lib/src/mt_ptp.c

```diff
@@ -500,7 +552,9 @@ static int ptp_parse_result(struct mt_ptp_impl* ptp) {
         ptp_result_reset(ptp);
       }
       ptp_sync_expect_result(ptp);
+#ifndef MTL_HAS_DPDK_TIMESYNC_ADJUST_FREQ
```

Comment:
Not prefer to reduce adjustment time with the real delta.

Reply:
Only issue is this will break the error handling for E810...

Reply:
> Only issue is this will break the error handling for E810...

Keep E810  error handling while phc2sys service is disabled.